tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -39,6 +39,7 @@ struct choose_literal<wchar_t> {
 
 #define TYPED_LITERAL(CharT, Literal) (choose_literal<CharT>::choose(Literal, L##Literal))
 #define STR(Str)                      TYPED_LITERAL(charT, Str)
+#define STR_VIEW(Str)                 basic_string_view(STR(Str))

```

Comment:
Style: We should use braces to construct the `basic_string_view` temporary here.

Reply:
If I try to use braces `clang-format` does this:
```
#define STR_VIEW(Str)          \
    basic_string_view<charT> { \
        STR(Str)               \
    }
```

Reply:
That's gross, yeah. I'm okay with parentheses then (it's test code, and it's a class type, so it won't perform a functional-style C-semantics cast). Thanks for looking into this!